apps/api/tests/api/application-state.test.ts

```diff
@@ -423,8 +445,6 @@ describe('State Machine', () => {
 	});
 
 	after(async () => {
-		await db.delete(applicationActions);
-		await db.delete(applications);
```

Comment:
Every test is using an individual test container so these `deletes` are likely just adding time and possibly a source of errors team members are encountering